### ISSUE TITLE
Coordinator verification takes long time

### ISSUE THREAD

**Issue by bosomt** [trezor/trezor-suite#7441]
**Describe the bug**
During last round of coinjoin i experienced `Coordinator verification...` almost whole time i was running coinjoin.
After some time it disappeared but it was back in short time.

Transactions were sent. But countdown i experienced made Trezor Suite unusable and i had to reload it to stop coinjoin.
Never experienced before and i removed remembered wallets before i started coinjoin.

**Steps to reproduce:**

**Info:**
 - Suite version: desktop 23.2.0 (47ff6c1b0068967d941c67675a9b6cc012464fb7)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuite/23.2.0 Chrome/104.0.5112.124 Electron/20.3.5 Safari/537.36
 - OS: MacIntel
 - Screen: 1440x900
 - Device: 
 - Transport: bridge 2.0.32

**Screenshots:**
![image](https://user-images.githubusercontent.com/31506317/214245137-7f4b01b0-05cd-4569-a030-ed78c6ada7fa.png)

[trezor-suite-log-2023-01-23T13-42-33.txt.zip](https://github.com/trezor/trezor-suite/files/10487689/trezor-suite-log-2023-01-23T13-42-33.txt.zip)

**Comment by szymonlesisz** [comment]
it's not a bug but expected behavior, coordinator instructs all registered users that they have 6 minutes to register their outputs.
Each wallet does that individually using random delays and order, at the end of output registration phase wallet confirms registered outputs using `/ready-to-sign` request. Whole process is divided in [time ranges](https://github.com/trezor/WalletWasabi/blob/master/WalletWasabi/WabiSabi/Client/CoinJoinClient.cs#L1057):
```
// Splitting the remaining time.
// Both operations are done under output registration phase, so we have to do the random timing taking that into account.
var outputRegistrationEndTime = now + (remainingTime * 0.8); // 80% of the time.
var readyToSignEndTime = outputRegistrationEndTime + remainingTime * 0.2; // 20% of the time.
``` 

in other words output registration takes almost 5 minutes, and remaining time is used to sent `ready-to-sing` 

until that suite doesnt have all the data to sign transaction, therefore needs to wait to continue

**Comment by bosomt** [comment]
Maybe we can consider to add possibility to close/hide this modal ? 
If its only idling and i have to wait to use Trezor Suite additional minutes.

**Comment by hynek-jina** [comment]
@bosomt 
We are considering two options (both for post-MVP phase)
- https://github.com/trezor/trezor-suite/issues/7498
- Remove this blocking modal in non-signing parts (but it opens a lot of questions)

**Comment by hynek-jina** [comment]
@szymonlesisz @bosomt 
So the times described here are expected?
So we can close the issue, right?
